### PR TITLE
[-] MO : Fixed correct display of tax label advancedeucompliance.php

### PR DESCRIPTION
**Pull request by Nobodaddy**
Correct display of tax label when tax excluded

### CODE REVIEW

**Comment by vTerenti** [comment]
Tested : OK :)

Can be merged

File: advancedeucompliance.php [hunk 1]
```diff
@@ -671,8 +672,9 @@ public function hookDisplayProductPriceBlock($param)
 
                 $customer_default_group_id = (int)$this->context->customer->id_default_group;
                 $customer_default_group = new Group($customer_default_group_id);
+                $display_tax_label = $this->context->country->display_tax_label;
 
-                if ((bool)Configuration::get('PS_TAX') === true &&
+                if ((bool)Configuration::get('PS_TAX') === true && ($display_tax_label) == true &&
```
Comment (maximebiloe) [inline advancedeucompliance.php]: Why do you use `($display_tax_label)` == true` instead of just `$display_tax_label` ?

**Comment by maximebiloe** [comment]
Hello @Nobodaddy,

Thanks for your contribution.
In order to merge your PR, could you amend your commit message to follow our [guidelines](http://doc.prestashop.com/display/PS16/How+to+write+a+commit+message), please ?

Best regards

File: advancedeucompliance.php (hunk 1 above)
Comment (Nobodaddy) [inline advancedeucompliance.php]: Simply because I just took some old code snippet from EU legal without a second thought. ;)

**Comment by maximebiloe** [comment]
Hello,

I see you changed your PR title but not the commit message. Could you amend it, please ?

Regards

**Comment by Nobodaddy** [comment]
How about you tell me exactly what you expect and how to do it? Because I don't get it.  Unlike you I'm no github expert. ;)

**Comment by maximebiloe** [comment]
To rewrite the commit message, you can look at [this](https://git-scm.com/book/en/v2/Git-Tools-Rewriting-History). Then just do a `push -f` to your branch.
If you're not sure, I'll do it for you for this time :wink:

**Comment by Nobodaddy** [comment]
Sending Unix commands from an Android tablet, seems to me a bit difficult 😊

**Comment by Nobodaddy** [comment]
I hate useless discussions. I close this pull request.

**Pull request closed without merging**